README.md

```diff
@@ -155,6 +152,30 @@ auto-generated. E.g. `bundle.yaml` originates from the _Jsonnet_ files in
 2. Run `make generate-in-docker`.
 3. Commit the generated changes.
 
+## How to add / remove Go dependencies
+
+This project uses [go module](https://github.com/golang/go/wiki/Modules).
+
+We do "vendoring" (committing dependencies code into project).
```

Comment:
I would replace this wording with something like this to avoid the "we":
```suggestion
For this project, the vendor directory is committed to the repository.
```